### PR TITLE
Added a content_settings stanza to the /status/ output.

### LINKED ISSUE #3138: Add the URL of `content-app` to the pulp's status endpoint
Users who execute `pulp status` shall see the URL of content-app where all resources are served by a distribution. This might be useful for determining the URL of the Pulp Registry when using the container plugin.

### PR DESCRIPTION
**Pull request by ggainey**
closes #3138.

### CODE REVIEW

File: pulpcore/tests/functional/api/test_status.py [hunk 1]
```diff
@@ -44,6 +44,10 @@
             },
         },
     },
+    "content_settings": {
+        "content_origin": "http://localhost:5001",
+        "content_path_prefix": "/pulp/content/",
+    },
```
Comment (lubosmj) [inline pulpcore/tests/functional/api/test_status.py:50]: This is not correct. It should contain JSON schema types, not constants.

Comment (lubosmj) [reply pulpcore/tests/functional/api/test_status.py:50]: Something like this:

```suggestion
    "content_settings": {
        "type": "object",
        "properties": {
            "content_origin":  {"type": "string"},
            "content_path_prefix": {"type": "string"},
        }
    },
```

Comment (ggainey) [reply pulpcore/tests/functional/api/test_status.py:50]: Ha, perfect! And this is why we do reviews :)

File: pulpcore/tests/functional/api/test_status.py [hunk 2]
```diff
@@ -60,6 +75,7 @@ class StatusTestCase(unittest.TestCase):
 
     def setUp(self):
         """Make an API client."""
+        self.cfg = config.get_config()
         self.client = api.Client(config.get_config(), api.json_handler)
         self.status_response = STATUS
         cli_client = cli.Client(config.get_config())
```
Comment (mdellweg) [inline pulpcore/tests/functional/api/test_status.py:81]:
```suggestion
        self.client = api.Client(self.cfg, api.json_handler)
        self.status_response = STATUS
        cli_client = cli.Client(self.cfg)
```

**Review by mdellweg — APPROVED** [review]

File: pulpcore/app/serializers/status.py [hunk 3]
```diff
@@ -49,6 +49,19 @@ class StorageSerializer(serializers.Serializer):
     free = serializers.IntegerField(min_value=0, help_text=_("Number of free bytes"))
 
 
+class ContentSettingsSerializer(serializers.Serializer):
+    """
+    Serializer for information about content-app-settings for the pulp instance
+    """
+
+    content_origin = serializers.CharField(
+        help_text=_("The CONTENT_ORIGIN setting for this Pulp instance"),
```
Comment (bmbouter) [inline pulpcore/app/serializers/status.py:58]:
```suggestion
        help_text=_("The CONTENT_ORIGIN setting value."),
```

**Review by bmbouter — APPROVED** [review]
Thanks!

**Pull request merged**